src/ILCompiler/reproNative/reproNative.vcxproj

```diff
@@ -20,7 +20,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <!-- v140 is friendly to windbg, at the moment -->
+    <PlatformToolset>v140</PlatformToolset>
```

Comment:
I'll submit a separate PR for this after a bit more investigation (not my intention to put this in). Someone was suggesting fastlink becoming default as a possibility (not sure if it is the same as back pointers to .obj files) but I haven't got a chance to run more experiment yet.
BTW, this only means VS 2015 toolset is required but you can still use VS 2017. There is still a annoying dialog to ask you to upgrade, though.